### PR TITLE
Add documentation for crossover functions

### PR DESCRIPTION
**Pull request by ArpitShukIa**
Added documentation for the following crossovers:
- [x] blend
- [x] cycle
- [x] linear
- [x] multi_point
- [x] order
- [x] partially_mapped
- [x] shuffle
- [x] simulated_binary
- [x] single_point
- [x] uniform
- [ ] uniform_partially_mapped

### CODE REVIEW

File: src/crossover/cycle.rs [hunk 1]
```diff
@@ -4,6 +4,24 @@ use itertools::multizip;
...
+
+## Example:
+```rust
+use genx::crossover::cycle_crossover;
+
+let parent1 = vec![1, 3, 4, 7, 8, 2, 6, 5];
+let parent2 = vec![2, 3, 4, 8, 7, 6, 1, 5];
+let (child1, child2) = cycle_crossover(&parent1, &parent2);
+```
+ */
```
Comment (king-11) [inline src/crossover/cycle.rs:24]: not cycle its multi-point actually that you have cited

**Review by king-11 — CHANGES_REQUESTED** [review]

**Comment by king-11** [comment]
You can find mapped ones in the paper and blog maybe add them too.

File: src/crossover/cycle.rs (hunk 1 above)
Comment (ArpitShukIa) [reply src/crossover/cycle.rs]: Fixed it.

**Comment by king-11** [comment]
LGTM 👍🏻

**Pull request merged**